### PR TITLE
[main][feat][alpaca-1954] AUSD deployment script

### PR DESCRIPTION
**Pull request by jakeal-papaca**
## Description
 - Deployment script

### CODE REVIEW

File: .testnet.json [hunk 1]
```diff
@@ -0,0 +1,92 @@
+{
+    "Timelock": "",
+    "ProxyWalletFactory": {
+        "address": "0xd065FD43827dc3289F089767397aaE13C77826f8"
```
Comment (ricardo-alpaca) [inline .testnet.json:4]: Add `deployedBlock`

File: deploy/exec/alpaca-stablecoin/config/grant-minter-role.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  ░██║░░██╗░░██║██╔══██╗██╔══██╗████╗░██║██║████╗░██║██╔════╝░
+  ░╚██╗████╗██╔╝███████║██████╔╝██╔██╗██║██║██╔██╗██║██║░░██╗░
+  ░░████╔═████║░██╔══██║██╔══██╗██║╚████║██║██║╚████║██║░░╚██╗
+  ░░╚██╔╝░╚██╔╝░██║░░██║██║░░██║██║░╚███║██║██║░╚███║╚██████╔╝
+  ░░░╚═╝░░░╚═╝░░╚═╝░░╚═╝╚═╝░░╚═╝╚═╝░░╚══╝╚═╝╚═╝░░╚══╝░╚═════╝░
+  Check all variables below before execute the deployment script
+  */
+
+  const ALPACA_STABLECOIN_ADDR = "0x64c12ac760463FF3B6849FC1ab22Fe8D1c6e06de"
+  const MINTER_ADDR = "0xea1FBaA5C061CF829Da88B3d711f7C2E4C09B9DC"
```
Comment (ricardo-alpaca) [inline deploy/exec/alpaca-stablecoin/config/grant-minter-role.ts:19]: We could say this is `STABLECOIN_ADAPTER_ADDR` to be specific. It should be the only one with `MINTER_ROLE`.

File: deploy/exec/proxy/proxy-actions/deploy/proxy-action.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  ░██╗░░░░░░░██╗░█████╗░██████╗░███╗░░██╗██╗███╗░░██╗░██████╗░
+  ░██║░░██╗░░██║██╔══██╗██╔══██╗████╗░██║██║████╗░██║██╔════╝░
+  ░╚██╗████╗██╔╝███████║██████╔╝██╔██╗██║██║██╔██╗██║██║░░██╗░
+  ░░████╔═████║░██╔══██║██╔══██╗██║╚████║██║██║╚████║██║░░╚██╗
+  ░░╚██╔╝░╚██╔╝░██║░░██║██║░░██║██║░╚███║██║██║░╚███║╚██████╔╝
+  ░░░╚═╝░░░╚═╝░░╚═╝░░╚═╝╚═╝░░╚═╝╚═╝░░╚══╝╚═╝╚═╝░░╚══╝░╚═════╝░
+  Check all variables below before execute the deployment script
+  */
+
+  console.log(">> Deploying an upgradable AlpacaStablecoinProxyAction contract")
```
Comment (ricardo-alpaca) [inline deploy/exec/proxy/proxy-actions/deploy/proxy-action.ts:18]: This contract is not upgradable.

File: deploy/exec/proxy/proxy-wallet/deploy/proxy-wallet-factory.ts [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  ░██╗░░░░░░░██╗░█████╗░██████╗░███╗░░██╗██╗███╗░░██╗░██████╗░
+  ░██║░░██╗░░██║██╔══██╗██╔══██╗████╗░██║██║████╗░██║██╔════╝░
+  ░╚██╗████╗██╔╝███████║██████╔╝██╔██╗██║██║██╔██╗██║██║░░██╗░
+  ░░████╔═████║░██╔══██║██╔══██╗██║╚████║██║██║╚████║██║░░╚██╗
+  ░░╚██╔╝░╚██╔╝░██║░░██║██║░░██║██║░╚███║██║██║░╚███║╚██████╔╝
+  ░░░╚═╝░░░╚═╝░░╚═╝░░╚═╝╚═╝░░╚═╝╚═╝░░╚══╝╚═╝╚═╝░░╚══╝░╚═════╝░
+  Check all variables below before execute the deployment script
+  */
+
+  console.log(">> Deploying an upgradable ProxyWalletFactory contract")
```
Comment (ricardo-alpaca) [inline deploy/exec/proxy/proxy-wallet/deploy/proxy-wallet-factory.ts:17]: not upgradable

**Review by ricardo-alpaca — APPROVED** [review]

**Review by almost-llamas — APPROVED** [review]

**Pull request merged**